packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -93,14 +93,24 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
     // TODO We may need to elevate classNames from <FocusZone> into <ResizeGroup> ?
     const nativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
 
+    // Given the same cache key, RenderedData will return the same memoized JSX Element,
+    // avoiding any re-renders of the control
+    const onRenderData = (props: ICommandBarData) => {
+      const compareData = (prevProps: ICommandBarData, nextProps: ICommandBarData) => {
+        return prevProps.cacheKey === nextProps.cacheKey;
```

Comment:
It looks like the cacheKey on ICommandBarData does not include the farItems prop, however the _onRenderData function seems to use that prop, it seems like we might need to include farItems on compareData (note: I'm not an expert on this component).

Reply:
i initially left far items out because they don't change over resize. The only time far items would change would be is user passed in new props. Guess the question is, would that invalidate the cache?